_extensions/quarto-svgbob/_extension.yml

```diff
@@ -1,6 +1,7 @@
 title: quarto-svgbob
 author: Dheepak Krishnamurthy
-version: 1.0.0
+  # - Fernando Martínez Santa
```

Comment:
Do you want to use the authors block here instead?

```
authors:
  - name: Dheepak Krishnamurthy
  - name: Fernando Martinez Santa
 ```

That way you'll get credit for it on the quarto website.

Reply:
sounds good..